live-site/components/judging-section/judging-schedule-welcome/JudgingScheduleWelcome.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { StyledBackButtonContainer, StyledBackButton, StyledJudgesDropdownContainer, StyledJudgesDropdownWrapper, StyledJudgingScheduleSection, StyledWelcomePerson } from '../judging-schedule-welcome/JudgingScheduleWelcome.styles';
+import { JudgingScheduleWelcomeProps } from '../../../lib/types';
+
+const JudgingScheduleWelcome: React.FC<JudgingScheduleWelcomeProps> = ({ schedulePersonType }) => {
+  const personOptions = ['Select your name'];
+
+  return (
+    <div>
+      <StyledBackButtonContainer> 
+        <StyledBackButton btnText='back' btnLink='../' /> 
```

Comment:
Like Mike said, this links back to the home page - we should conditionally set the btnLink based on the schedulePersonType

Reply:
works correctly for me? we'd like it to go back to judging selection page regardless of personType which this handles appropriately right now!